datanode/gateway/graphql/schema.graphql

```diff
@@ -617,8 +617,7 @@ type Query {
   orderByID(
     "ID for an order"
     id: ID!
-
-    "Version of the order (omitted or 0 for most recent; 1 for original; 2 for first amendment, etc)"
+    "Order version number, starting at 1 for the original order and incrementing by 1 for each successful amendment (omitted for the most recent version)."
```

Comment:
```suggestion
    "Order version number, starting at 1 for the original order and incrementing by 1 for each successful amendment; omitted for the most recent version."
```

Just removing the brackets here, otherwise good.